packages/block-library/src/image/edit.js

```diff
@@ -339,17 +353,8 @@ class ImageEdit extends Component {
 	}
 
 	getImageSizeOptions() {
-		const { imageSizes, image } = this.props;
-		return compact( map( imageSizes, ( { name, slug } ) => {
-			const sizeUrl = get( image, [ 'media_details', 'sizes', slug, 'source_url' ] );
```

Comment:
Do you know why this was removed as well. I'm assuming sometimes, images can be uploaded before the image size being defined which causes the size to not be available for that particular image.

Reply:
Oh I see that you moved the check to the "updateImage" function

Reply:
I wonder if the user could get confused if using the "select" doesn't change anything. Not critical for the moment though.